FV3/wrapper/templates/_wrapper.pyx

```diff
@@ -62,16 +63,18 @@ cdef extern:
 {% endfor %}
 
 cdef get_quantity_factory():
-    cdef int nx, ny, nz, nz_soil
+    cdef int nx, ny, nz, nz_soil, n_topo_variables
     get_centered_grid_dimensions(&nx, &ny, &nz)
     get_nz_soil_subroutine(&nz_soil)
+    get_n_topo_variables_subroutine(&n_topo_variables)
     sizer = pace.util.SubtileGridSizer(
         nx,
         ny,
         nz,
         n_halo=pace.util.N_HALO_DEFAULT,
         extra_dim_lengths={
             pace.util.Z_SOIL_DIM: nz_soil,
+            pace.util.N_TOPO_DIM:n_topo_variables,
```

Comment:
You need to define this variable. A hard-coded name is probably good for now. Make sure to leave a whitespace after the `:`.

Reply:
```suggestion
'orographic_variable': n_topo_variables,
```